packages/cna-template/template/tsconfig.json

```diff
@@ -25,7 +25,12 @@
     },
     "types": [
       "@types/node",
+      <%_ if (content) { _%>
+      "@nuxt/types",
+      "@nuxt/content"
+      <%_ } else { _%>
       "@nuxt/types"
+      <%_ } _%>
```

Comment:
```suggestion
      "@nuxt/types",
      <%_ if (content) { _%>
      "@nuxt/content",
      <%_ } _%>
      "@types/node"
```

Reply:
Hmm. I was under impression the order would matter here, but doesn't seem to matter... especially since nuxt would not extend node